phd-tests/framework/src/artifacts/store.rs

```diff
@@ -0,0 +1,288 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use crate::guest_os::GuestOsKind;
+
+use super::{manifest::Manifest, ArtifactSource};
+use anyhow::bail;
+use camino::{Utf8Path, Utf8PathBuf};
+use ring::digest::{Digest, SHA256};
+use std::collections::BTreeMap;
+use std::fs::File;
+use std::io::{BufReader, Read, Seek, SeekFrom, Write};
+use std::sync::Mutex;
+use std::time::Duration;
+use tracing::info;
```

Comment:
small style nit: group the crate-local (`crate::` and `super::`), `std`, and cargo imports together in their own sections to stay consistent the rest of the existing style?